tensorcircuit/channels.py

```diff
@@ -127,49 +145,28 @@ def generaldepolarizingchannel(
 
     if num_qubits == 1:
         if isinstance(p, float):
-            assert p > 0 and p < 1 / 3, "p should be >0 and <1/3"
             probs = [1 - 3 * p] + 3 * [p]
-
         elif isinstance(p, list):
```

Comment:
or isinstance(p, tuple)?

Reply:
fixed in the next commit. BTW the original implementation for tuple conversion is
```
p = list[p]
```
which makes `p` a `type` object